New What's New for 0.8.2.

Racel opened the pull request:
### Purpose

Updated What's New Pages for 0.8.2. Please take a look and let me know if any changes need to be made.

### Declarations

Check these if you believe they are true

- [ ] The code base is in a better state after this PR

### Reviewers
@ikeough 
@Benglin 
@RodRecker 
@riteshchandawar 

### FYIs
@jnealb

lukechurch commented on extern/gallery/en-US/GalleryContents.xml:
```diff
@@ -2,29 +2,34 @@
-		<Body>All your working files are now safe even when Dynamo crashes. Look out for recovered files in the start page when you restart Dynamo after a crash.</Body>
-		<ImagePath>crash_recovery.png</ImagePath>
+		<Header>Convert Nodes to Code</Header>
+		<Body>Select groups of nodes and automatically generate the corresponding Design Script code.</Body>
```
DesignScript is one word :)

RodRecker commented:
@Racel Can you add a note about command line usage? Maybe add it on the last page.

holyjewsus commented:
@RodRecker @Racel the documentation is here on the wiki
https://github.com/DynamoDS/Dynamo/wiki/Dynamo-Command-Line-Interface

Racel commented:
@RodRecker -Added a line about it on the last page.

RodRecker commented:
@Racel looks good - thank you!

Racel commented:
@RodRecker - Made a minor change after some discussion with @holyjewsus. Let me know if I can go ahead and merge with master.

Benglin commented:
Thanks for the work, @Racel, it looks good to me. When it comes to testing, I'd be interested to see how the new installer remove older gallery files (e.g. `%ProgramData%\Dynamo\0.8\gallery\Data\multiple_previews.png`). I'm sure @jnealb or @riteshchandawar can see if that works.

Benglin commented:
On a second thought, these images contain various text contents in them, which means they will look odd on other languages. there are two ways around this:

1. Move the images into `extern/gallery/en-US` folder, and they will be localized by WWL team.
2. Update images to contain lesser (and language agnostic contents) or no text,

riteshchandawar commented:
@Racel Publish Local is missing for what's new section.
![image](https://cloud.githubusercontent.com/assets/5109531/9299147/621fad32-44eb-11e5-9608-5e1a296429d8.png)


In-Canvas search was delivered in 0.8.1 as well. We need to showcase now In-Canvas Search have capabilities of Tooltip and keyboard navigation keys.

Thanks,
Ritesh

Racel commented:
@Benglin @riteshchandawar:
 - I moved the images to extern/gallery/en-US for localization. FYI - previous What's New slides contain English text also, though. I used current slides as a template.
 - Added some text to the 4th slide indicating improved keyboard navigation for search
 - Added image/text related to Publish Local

Please let me know if it is ok now.

RodRecker commented:
@monikaprabhu Can you check that the localization team can handle these images?

ikeough commented on extern/gallery/en-US/GalleryContents.xml:
```diff
@@ -2,29 +2,34 @@
-		<Body>You can now drag items from the library. It helps you place nodes precisely on the first attempt.</Body>
-		<ImagePath>library_draggable.png</ImagePath>
+		<Header>Color on Surfaces</Header>
+		<Body>Smooth color shading can now be applied to surfaces with the Display.BySurfaceColors node.</Body>
```
We should not use the term "Smooth color shading". In 3D, smooth shading is shading that incorporates the surface normal into its coloration calculations. This is in contrast to flat shading which only considers the triangle normal.

Suggest:
"Square arrays of color can now be applied..."

Racel commented:
Thanks, @ikeough. I updated the text and added another sentence about applying images. Please take a look.

Racel commented:
@Benglin - can you tell me how to change '0.8' at the top of the what's new template to '0.8.2'?

Benglin commented:
@Racel, I figured it takes less time for me to code/test it than to put a description here making sure things are covered. You can cherry-pick [this commit from my repository](https://github.com/DynamoDS/Dynamo/commit/009464db12f1f175602263bec9a7ba786ac203d5) instead :)

Racel commented:
@Benglin - 
Thanks for the commit. We will cherry-pick that.

@ikeough @RodRecker @Benglin @riteshchandawar - Updated the images based off of @Benglin's suggestions. The text in the images has been reduced for better localization practices. 

@ikeough - got rid of the extra sentence in the 'Color on Surfaces' slide.

Let me know if this is ready for cherry-picking. We will need to grab this and @Benglin's commit above.

ikeough commented:
LGTM. Thanks @Racel!

Racel commented:
@Benglin - added your commit to this PR. Did not want to have to cherry pick them separately.

riteshchandawar commented:
@lukechurch @Racel We need to cherry pick this to RC 082 brach as well.

- Ritesh

Racel commented:
@riteshchandawar - we cherry picked it for the RC 082 yesterday. The commit on RC 082 includes ben's commit.

riteshchandawar commented:
Thanks @Racel

Benglin commented:
I like your new images very much, @Racel, awesome stuff! :+1:
